src/components/side_panel/chart/scorecard_chart_panel/scorecard_chart_design_panel.ts

```diff
@@ -73,4 +86,54 @@ export class ScorecardChartDesignPanel extends Component<Props, SpreadsheetChild
         break;
     }
   }
+
+  get keyStyle(): TitleDesign {
+    return {
+      align: "center",
+      fontSize: DEFAULT_SCORECARD_KEY_VALUE_FONT_SIZE,
+      ...this.props.definition.keyDescr,
+    };
+  }
+
+  get baselineStyle(): TitleDesign {
+    return {
+      align: "center",
+      fontSize: DEFAULT_SCORECARD_BASELINE_FONT_SIZE,
+      ...this.props.definition.baselineDescr,
+    };
+  }
+
+  updateKeyStyle(key: string, value: any) {
```

Comment:
avoid `any` as much as possible. It always back fires sooner or later.
In this case, I'd do one method per style property, and use each one at the right place in the template.
It might be a bit of boilerplate (although the switch/case also is) but at least the TS part is typed properly and we are not mixing different things together.